packages/website/pages/blog/index.js

```diff
@@ -0,0 +1,319 @@
+import { Card, HighlightCard } from '../../components/blog/cards'
+import { useEffect, useState } from 'react'
+
+import Button from '../../components/button'
+import Loading from '../../components/loading'
+import Tags from '../../components/tags'
+import { allTags } from '../../components/blog/constants'
+import clsx from 'clsx'
+import fs from 'fs'
+import matter from 'gray-matter'
+import { usePagination } from '../../lib/usePagination'
+import { useRouter } from 'next/router'
+
+const BLOG_ITEMS_PER_PAGE = 9
+
+export async function getStaticProps() {
+  const files = fs.readdirSync('all-blogs')
+
+  let featuredImage = null
+
+  // we can create a "featured" flag if we want and feature blogs that way
+
+  /**
+   * @param {string} fn
+   */
+  const getBirthtime = (fn) =>
+    fs.statSync(`all-blogs/${fn}`).birthtime.getTime()
```

Comment:
Does this work if I freshly clone the repo? Don't all the files get the same birthtime at that point? Personally, I'd use a convention of `YYYY-MM-DD-post-name.mdx` and sort alphabetically.

You also don't need a `date` in the front matter.

Reply:
I believe it works; didn't know at the time if I should assume naming conventions. I'll change to the way you suggested. 

I also thought that we might want to add a flag because we might want to highlight a blog that isn't the newest. Can always change in the future if desired. 

@the-simian also suggested we could do a carousel if we wanted to highlight multiple blogs.